addons/getLoadoutsFromUnits/functions/fnc_getItems.sqf

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+params ["_items", "_breakline" , "_tab"];
+
+private _return = "";
+private _count = count _items;
+if (_count > 0) then {
+   private _quadrupleTab = _tab + _tab + _tab + _tab;
+   {
+       if (count _x == 3) then {
+          _return = _return + _breakline + _quadrupleTab + format ["LIST_%1('%2'),", _x select 2, _x select 0];
+       }else{
+          _return = _return + _breakline + _quadrupleTab + format ["LIST_%1('%2'),", _x select 1, _x select 0];
```

Comment:
you are setting _return for each item and only using the last one. So you only want the last item out of _items? Why not just take the last then instead of iterating over all the items that you don't want?
Also you could just _return = {...} forEach _items; to get the last value that was returned inside the forEach code segment.

Reply:
No, you are missunderstanding something here, I want all Items. they are just stacked together.